02week/nasa/home.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>NASA</title>
+    <link href="https://fonts.googleapis.com/css?family=Black+Ops+One|Roboto" rel="stylesheet">
+    <link rel="stylesheet" href="nasastyle.css">
+</head>
+
+<body>
+    <main>
+        <section id="background">
+            <h1 id="header">nasa</h1>
+        </section>
+
+        <section id="grey">
+            <article class="info">
+                <h1>Social Media</h1>
+                <p>
+                    Twitter - @nasa<br> Facebook - fb.com/nasa<br> Instagram - @nasa<br>                    Youtube - youtube.com/NASA
```

Comment:
Instead of inserting breaks between each item of information, it's easier to just make each a <p> of its own. I think that should make it easier in terms of spacing in the future. I think.